rd_ui/app/scripts/visualizations/boxplot.js

```diff
@@ -0,0 +1,184 @@
+(function() {
+  var module = angular.module('redash.visualization');
+
+  module.config(['VisualizationProvider', function(VisualizationProvider) {
+      var renderTemplate =
+        '<boxplot-renderer ' +
+        'options="visualization.options" query-result="queryResult">' +
+        '</boxplot-renderer>';
+
+      var editTemplate = '<boxplot-editor></boxplot-editor>';
+      var defaultOptions = {
+        boxplotColName: 'boxplot',
+        rowNumber: 1,
+        targetRowNumber: 1
+      };
+
+      VisualizationProvider.registerVisualization({
+        type: 'BOXPLOT',
+        name: 'Boxplot',
+        renderTemplate: renderTemplate,
+        editorTemplate: editTemplate,
+        defaultOptions: defaultOptions
+      });
+    }
+  ]);
+  module.directive('boxplotRenderer', function() {
+    return {
+      restrict: 'E',
+      templateUrl: '/views/visualizations/boxplot.html',
+      link: function($scope, elm, attrs) {
+
+        
+        var loaddata = function(inputdata){
```

Comment:
You are correct! It was used while I was developing and I guess it managed to hang around. 
Fixed in 0b5dce0